tests/test_wizard_postgres.py

```diff
@@ -87,3 +95,84 @@ def test_postgres():
     assert 'title' in str(ref_list)
     assert 'url' in str(ref_list)
     assert 'research-hub.nrel.gov' in str(ref_list)
+
+
+def test_ref_replace(mocker):
+    """Test to ensure removal of double quotes from references."""
+    mock_conn_cm = mocker.MagicMock()
+    mock_conn = mock_conn_cm.__enter__.return_value
+    mock_conn.cursor.return_value = Cursor()
+
+    mock_connect = mocker.patch('psycopg2.connect')
+    mock_connect.return_value = mock_conn_cm
```

Comment:
Do you need the mock setup here? I don't think `_format_refs` does any database calls, right? So I think you can just remove this from this test and the subsequent ones

Reply:
The way I had it set up initially I needed to mock the db connection to initialize `self.cursor`. I am passing the mock cursor class now to avoid that.